membership-attribute-service/conf/TEST.public.conf

```diff
@@ -0,0 +1,59 @@
+include "touchpoint.DEV.conf"
```

Comment:
I added a custom test configuration (just copied from application.conf and dev.public.conf) otherwise the tests did not pass in team city. I'm not sure if this is the most elegant solution but it works